.travis.yml

```diff
@@ -1,47 +1,34 @@
 language: php
-
-addons:
-  apt:
-    packages:
-      - tidy
-
-php:
-  - 5.6
-  - 7.0
-  - 7.1
-
-before_install:
-  - pip install --user codecov
-
-env:
-  global:
-    - DB=MYSQL CORE_RELEASE=4
+dist: precise
 
 matrix:
   include:
-    - php: 7.1
-      env: DB=MYSQL CORE_RELEASE=4 COVERAGE=1
     - php: 5.6
-      env: DB=PGSQL CORE_RELEASE=4
-    - php: 5.6
-      env: DB=SQLITE CORE_RELEASE=4
+      env: DB=MYSQL PHPCS_TEST=1 PHPUNIT_TEST=1
+    - php: 7.0
+      env: DB=PGSQL PHPUNIT_TEST=1
+    - php: 7.1
+      env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1
+    - php: 7.0
+      env: DB=MYSQL
 
 before_script:
   - phpenv rehash
-  - composer self-update || true
-  - git clone git://github.com/silverstripe/silverstripe-travis-support.git ~/travis-support
-  # Install suggested modules in order to maximize test coverage
-  - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss --require "ezyang/htmlpurifier:4.*,silverstripe/cms:^4.0@dev"
-  - cd ~/builds/ss
+  - phpenv config-rm xdebug.ini
+  - echo 'memory_limit = 2G' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+  - composer validate
+  - composer require silverstripe/recipe-cms 1.0.x-dev --no-update
+  - if [[ $DB == PGSQL ]]; then composer require --no-update silverstripe/postgresql:2.0.x-dev; fi
+  - composer install --prefer-dist --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
+  - echo "SS_BASE_URL=http://localhost:8080/" >> .env
 
-script:
-  # Execute tests with no coverage.  This is the fastest option
-  - "if [ \"$COVERAGE\" != \"1\" ]; then vendor/bin/phpunit comments/tests/; fi"
+  # Bootstrap cms / mysite folder
+  - php ./cms/tests/bootstrap/mysite.php
 
-  # Execute tests with coverage.  Do this for a small
-  - "if [ \"$COVERAGE\" = \"1\" ]; then vendor/bin/phpunit --coverage-clover=coverage.clover comments/tests/; fi"
+script:
+  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit; fi
```

Comment:
You need to pass the directory with the tests in

Reply:
Ah.. I was basing it off https://github.com/silverstripe/silverstripe-multiform/blob/master/.travis.yml#L30-L35 which doesn't seem to pass anything in unless I'm missing some flag

Reply:
It's fine to omit as long as it's defined in `phpunit.xml.dist`